### DISCUSSION TITLE
Set workspace title/label font or color.

### DISCUSSION THREAD

**Discussion by laowalter** [i3/i3#5601] [Q&A]
I would like to customize the colors and fonts of my workspace titles/labels. I have read the i3 documentation on how to do this,Have read https://i3wm.org/docs/userguide.html#client_colors,  but I have not been successful. Could someone provide an example of how to change the background color of unfocused workspace titles/labels?
Thanks.

**Comment by CodeWithMa** [comment]
Take a look at this nice website: https://thomashunter.name/i3-configurator/

You can adjust the colors of your bar in the `bar` block.

```
bar {
  colors {
    background #000000
    statusline #FFFFFF
    separator  #666666

    focused_workspace  #4C7899 #285577 #FFFFFF
    active_workspace   #333333 #222222 #FFFFFF
    inactive_workspace #333333 #222222 #888888
    urgent_workspace   #2F343A #900000 #FFFFFF
    binding_mode       #2F343A #900000 #FFFFFF
  }
}
```

**Reply by laowalter** [reply]
Thanks, got it.
